src/main/java/com/conleos/common/FormUtil.java

```diff
@@ -25,10 +25,10 @@ public static Span createFormBadge(Form form) {
             case Rejected -> badge.getElement().setAttribute("theme", "badge error");
         }
         switch (form.getStatus()) {
-            case InProgress -> badge.setText("In Progress");
-            case InReview -> badge.setText("In Review");
-            case Signed -> badge.setText("Signed");
-            case Rejected -> badge.setText("Rejected");
+            case InProgress -> badge.setText(Lang.translate("view.formUtil1"));
+            case InReview -> badge.setText(Lang.translate("view.formUtil2"));
+            case Signed -> badge.setText(Lang.translate("view.formUtil3"));
+            case Rejected -> badge.setText(Lang.translate("view.formUtil4"));
```

Comment:
view.formUtil1 is not a great placeholder for something like "In Progress"

Reply:
change：
view.formUtil.progress = In Progress
view.formUtil.review = In Review
view.formUtil.sign = Signed
view.formUtil.reject = Rejected